x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots/get_element_position_data.ts

```diff
@@ -4,48 +4,55 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { HeadlessChromiumDriver as HeadlessBrowser } from '../../../../server/browsers/chromium/driver';
+import { HeadlessChromiumDriver as HeadlessBrowser } from '../../../../server/browsers';
```

Comment:
This one, and the next 3 files have a lot of whitespace changes enforced by prettier. It would help to ignore whitespace changes in this PR review :)